dbfit-java/core/src/main/java/dbfit/util/TypeNormaliserFactory.java

```diff
@@ -1,50 +1,13 @@
 package dbfit.util;
 
-import java.util.HashMap;
-import java.util.Map;
-
-@SuppressWarnings("unchecked")
 public class TypeNormaliserFactory {
-    private static Map<Class, TypeNormaliser> normalisers = new HashMap<Class, TypeNormaliser>();
-
-    private static Class getCandidateIfAncestor(Class target, Class candidate) {
-        return candidate.isAssignableFrom(target) ? candidate : null;
-    }
-
-    private static Class getCandidateIfBetter(Class currentBest, Class nextCandidate) {
-        return currentBest.isAssignableFrom(nextCandidate) ? nextCandidate : currentBest;
-    }
-
-    private static Class findClosestAncestor(Class targetClass) {
-        Class currentBest = null;
+    private static TypeTransformerFactory ttf = new TypeTransformerFactory();  
```

Comment:
I got wondering if we can get rid of the global static state for that factory too. (I'm not sure yet if this PR is the place. Possibly yes.)